x-pack/legacy/plugins/task_manager/task_manager.ts

```diff
@@ -76,6 +69,15 @@ export class TaskManager {
     this.definitions = {};
     this.logger = opts.logger;
 
+    const taskManagerId = opts.config.get('server.uuid');
+    if (!taskManagerId) {
+      this.logger.error(
+        `TaskManager is unable to start as there the Kibana UUID is invalid (value of the "server.uuid" configuration is ${taskManagerId})`
+      );
+    } else {
```

Comment:
It's logging as unable to start, but the code still carries on as normal and as if the ID is valid.

Maybe throw an error here?